postman/metadata.csv

```diff
@@ -0,0 +1,15 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+PostmanMonitor_errors,gauge,,,,No of errors in an API run,0,Postman,
```

Comment:
Metrics usually follow the following convention: <integration_name>.<metric> and are all in lowercase

Reply:
Hi @hithwen thanks for pointing it out. Unfortunately, these metrics were implemented before we decided to create this integration. There are users who already have these metrics in production and hence renaming them might not be possible. 
Is it fine to keep this naming for the metrics that we have currently and follow the <integration_id>.<metric> for the new metric that we add.

Reply:
@alexvetras can you help here?